syscalls.c

```diff
@@ -1113,12 +1287,21 @@ int syscalls_sys_accept(char *ustack)
 	GETFROMSTACK(ustack, struct sockaddr *, address, 1);
 	GETFROMSTACK(ustack, socklen_t *, address_len, 2);
 
+	if (vm_mapBelongs(proc, address, sizeof(*address)) < 0) {
```

Comment:
`address` is optional. If not NULL it's size should be passed in `*address_len`.